src/frontend/next/src/components/Posts/Timeline.tsx

```diff
@@ -0,0 +1,57 @@
+import { Container, createStyles, Grid } from '@material-ui/core';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import PostComponent from '../Post';
+import { Post } from '../../interfaces';
+import Spinner from '../Spinner';
+import LoadAutoScroll from './LoadAutoScroll';
+import useSiteMetaData from '../../hooks/use-site-metadata';
+
+type Props = {
+  pages: Post[][] | undefined;
+  nextPage: Function;
+};
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      padding: 0,
+      maxWidth: '785px',
+    },
+    activeCircle: {
+      borderRadius: '4rem',
+      transition: 'all linear 250ms',
+      color: theme.palette.primary,
+    },
```

Comment:
@rogercyyu : This should be `theme.palette.primary.main`. I know the original file has it this way but TS complained about it I believe

Reply:
good catch